.zappr.yaml

```diff
@@ -1 +1,17 @@
 X-Zalando-Type: tools
```

Comment:
it should not be `tools` I guess

Reply:
should be "code"? I noticed a lot of team either using "tools" or not including this type at all.

Reply:
I think "code" should be ok.